<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaleManagerDialog</class>
 <widget class="QDialog" name="SaleManagerDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>675</width>
    <height>364</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>13</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Manage Sales</string>
  </property>
  <widget class="QPushButton" name="saveNewSaleButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>240</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="createButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>240</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Create</string>
   </property>
  </widget>
  <widget class="Q3ListBox" name="saleListBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>210</width>
     <height>180</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QLabel" name="saleLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>111</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Sale Dates:</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="closeSaleButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>130</y>
     <width>140</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Close Selected</string>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="saleDatePicker">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>40</y>
     <width>264</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="openSelected">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>190</y>
     <width>140</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Open Selected</string>
   </property>
  </widget>
  <widget class="QPushButton" name="setActiveButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>70</y>
     <width>140</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Set Active</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3Frame</class>
   <extends>QFrame</extends>
   <header>Qt3Support/Q3Frame</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3ListBox</class>
   <extends>Q3Frame</extends>
   <header>q3listbox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>saleListBox</tabstop>
  <tabstop>exitButton</tabstop>
  <tabstop>createButton</tabstop>
 </tabstops>
 <includes>
  <include location="local">dao/salemanager.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
